executor-service/src/main/java/executor/service/model/WebDriverConfig.java

```diff
@@ -1,11 +1,20 @@
 package executor.service.model;
 
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
 import java.util.Objects;
 
+@Component
+@PropertySource("classpath:config.properties")
 public class WebDriverConfig {
```

Comment:
Similarly, with `WebDriverConfig` we can put into one folder config.
and add `@ConfigurationProperties(prefix = "web-driver")`

Reply:
done, thanks